utils/AsyncStoageUtils.ts

```diff
@@ -1,5 +1,7 @@
 export const STORAGE_KEYS = {
-  start_day: () => "StartDay",
+  startDay: () => "StartDay",
   daysEpisodes: (recordingDay: number) => `Day${recordingDay}Episodes`,
   episodeRecall: () => "EpisodeRecall",
+  episodePredictions: (recordingDay: number) =>
```

Comment:
I am gonna revert this since we are never going to actually store episode predictions in storage

Reply:
Just a little bump to remember to do this